src/inputs/WizStringsInput.tsx

```diff
@@ -21,7 +21,7 @@ export type WizStringsInputProps = InputCommonProps & {
 export function WizStringsInput(props: WizStringsInputProps) {
     const { displayMode: mode, value, setValue, id, hidden } = useInput(props)
 
-    let values: string[] = value
+    let values: string[] = !value || Array.isArray(value) ? value : [value]
     if (!values) values = []
```

Comment:
```suggestion
      const values: string[] = Array.isArray(value) ? value : []
```
I'm not sure that putting the value into an array is a good idea, because then you get a text input for that value, but it is empty because the text input is still trying to read index 0 from the array.

Reply:
I'm assuming the `[]` value is preserved when the user hits "Submit" and the other invalid stuff that was entered is discarded or would it attempt to apply the invalid stuff and return with a validation error? I'm curious because this way does seem better/more accurate but in the other form, the user is prompted with a validation error up front as soon as they hit "Next". In the current way, the value stays visible in the YAML and without validation errors.

Reply:
Right. You won't get the error until you submit. What's in the YAML is what gets submitted. You might consider improving the AJV schema to validate more fully - then if the user edits the YAML they will at least see this is supposed to be an array. https://github.com/stolostron/console/blob/main/frontend/src/routes/Governance/policies/schema.json